components/FlexibleHeader/src/MDCFlexibleHeaderView.h

```diff
@@ -20,6 +20,27 @@ typedef void (^MDCFlexibleHeaderChangeContentInsetsBlock)(void);
 typedef void (^MDCFlexibleHeaderShadowIntensityChangeBlock)(CALayer *_Nonnull shadowLayer,
                                                             CGFloat intensity);
 
+/** This determines how we adjust the height to account for the Safe Area insets. */
+typedef NS_ENUM(NSInteger, MDCFlexibleHeaderHeightSafeAreaAdjustment) {
+
+  /**
+   Only adjust the height to account for the Safe Area if min and max have not been explicitly set.
+
+   In this mode we add safeAreaInsets.top to both the min and the max height only if they haven't
+   been explicitly set. Previously minimumHeight and maximumHeight could've included the status bar
+   height, so for backwards compatibility reasons we only adjust if the default values are being
+   used.
+   */
+  MDCFlexibleHeaderHeightSafeAreaAdjustmentLegacy,
```

Comment:
I think legacy doesn't explain what the enum is doing. Suggest rename to static or None.

Reply:
Static or None would mean that we never adjust, which is not true. If you don't change the default values we do adjust. Ian proposed Legacy, but maybe we could do something like MDCFlexibleHeaderSafeAreaAdjustsMinMaxHeightOnlyForDefaults?